app.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "MeiliSearch",
+  "description": "Instant search API",
+  "keywords": [
+    "productivity",
+    "HTML5",
+    "scalpel"
+  ],
+  "website": "https://www.meilisearch.com/",
```

Comment:
Do we redirect to the main website or to the doc?

Reply:
Maybe the doc would be better